specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/examples/RetrieveBootDiagnosticsDataVirtualMachine.json

```diff
@@ -0,0 +1,17 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "ResourceGroup",
+    "api-version": "2020-06-01",
+    "vmName": "VMName",
+    "sasUriExpirationTimeInMinutes": 60
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "consoleScreenshotBlobUri": "https://storageuri/vm.screenshot.bmp?sv=2017-04-17&sr=b&sig=hcanChQRWQIOzXddctW0KMI6vidtGYPTlXKhEpUaB40%3d&se=2020-03-02T23:01:12Z&sp=r",
+        "serialConsoleLogBlobUri": "https://storageuri/vm.serialconsole.log?sv=2017-04-17&sr=b&sig=hcanChQRWQIOzXddctW0KMI6vidtGYPTlXKhEpUaB40%3d&se=2020-03-02T23:01:12Z&sp=r"
```

Comment:
Let us confirm to the standard to show samples of SAS Uris, hopefully the sig param is not based on some existing resource

Reply:
@avinashakka  look at the example in CreateOrUpdateASimpleGalleryApplicationVersion.json, the format used to provide a sample sas uri should be something like https://storageuri/vm.screenshot.bmp?{sasKey}